samples/snippets/create_assessment.py

```diff
@@ -0,0 +1,80 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START recaptcha_enterprise_create_assessment]
+from google.cloud import recaptchaenterprise_v1
+
+
+def create_assessment(
+    project_id: str, recaptcha_site_key: str, token: str, recaptcha_action: str
+) -> None:
+    """ Create an assessment to analyze the risk of a UI action.
+    Args:
+        project_id: GCloud Project ID
+        recaptcha_site_key: Site key obtained by registering a domain/app to use recaptcha services.
+        token: The token obtained from the client on passing the recaptchaSiteKey.
+        recaptcha_action: Action name corresponding to the token.
+    """
+
+    # TODO(developer): Replace these variables before running the sample.
+    # Specify a name for this assessment.
+    assessment_name = "assessment_name"
+
+    client = recaptchaenterprise_v1.RecaptchaEnterpriseServiceClient()
+
+    # Set the properties of the event to be tracked.
+    event = recaptchaenterprise_v1.Event()
+    event.site_key = recaptcha_site_key
+    event.token = token
+
+    assessment = recaptchaenterprise_v1.Assessment()
+    assessment.event = event
+    assessment.name = assessment_name
+
+    project_name = f"projects/{project_id}"
+
+    # Build the assessment request.
+    request = recaptchaenterprise_v1.CreateAssessmentRequest()
+    request.assessment = assessment
+    request.parent = project_name
+
+    response = client.create_assessment(request)
+
+    # Check if the token is valid.
+    if not response.token_properties.valid:
+        print(
+            "The CreateAssessment call failed because the token was "
+            + "invalid for for the following reasons: "
+            + str(response.token_properties.invalid_reason)
+        )
+    else:
+        # Check if the expected action was executed.
+        if response.token_properties.action == recaptcha_action:
+            # Get the risk score and the reason(s)
+            # For more information on interpreting the assessment,
+            # see: https://cloud.google.com/recaptcha-enterprise/docs/interpret-assessment
+            for reason in response.risk_analysis.reasons:
+                print(reason)
+            print(
+                "The reCAPTCHA score for this token is: "
+                + str(response.risk_analysis.score)
+            )
+        else:
```

Comment:
nit: same here, but reversed - it's easier to read code if it's
```
if error
    # handle error
    return
# normal flow
``` 
as opposed to
```
if valid:
   # normal flow
else:
  # error handling